gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -492,8 +502,8 @@ def main(args):
         version=args.version,
     )
 
-    # The dbsnp table does not have a global field for dbsnp_versions, same
-    # with sift/polyphen and vrs (still need these)
+    # The dbsnp table does not have a global field for dbsnp_versions, same with
+    # sift/polyphen and vrs (still need these).
     ht = ht.annotate_globals(
```

Comment:
Should we add the Gencode version that is associated with the vep version?

Reply:
I think we should, it was even confusing to me at the beginning.

Reply:
I also want to add mane_select and mane_pluls_clinical version inside vep_globals. I tried very hard to convince Matt to use the same version as VEP 105 on the browser.

Reply:
Works for me, vep has its own global struct, do we think it belongs in there or down here?

Reply:
@KoalaQin, are you able to provide the versions for the tools mentioned in this thread?

Reply:
Yes, gencode_version="Release 39", mane_select_version="v0.95".

Reply:
They are coming from the same table, I think one is enough. only 56 are transcripts of 18640 are MANE Plus Clinical, it means that 56 genes have both MANE Select and MANE Plus Clinical. It seems we are not showing 'mane_pluls_clinical' on the browser, and for other in silico predictors, we only get score for the MANE Select transcripts.

Reply:
Or should we just use "mane_version"?

Reply:
The Ensembl people told me this: 
> The MANE Plus Clinical set includes additional transcripts for genes where
MANE Select alone is not sufficient to report all "Pathogenic (P)" or "Likely
Pathogenic (LP)" clinical variants available in public resources.

Reply:
hmm, I think mane_select is sufficient then. Thank you for all the context, this is helpful!